src/ui/public/directives/__tests__/timepicker.js

```diff
@@ -22,9 +22,6 @@ const init = function () {
   // Load the application
   ngMock.module('kibana');
 
-  // Stub out the clock so 'now' doesn't move
-  sinon.useFakeTimers(moment(anchor).valueOf());
```

Comment:
__note:__ It feels that it'd be better to keep `useFakeTimers` inside `beforeEach` instead of burying it somewhere inside custom `init` helper.